src/string.rs

```diff
@@ -0,0 +1,127 @@
+
+use bwapi_sys as sys;
+use from_raw::FromRaw;
+
+use std::ffi::CStr;
+use std::fmt;
+use std::ops::Deref;
+use std::os::raw::c_void as void;
+
+pub struct BwString(*mut sys::BwString);
+
+impl BwString {
+    pub fn len(&self) -> i32 {
+        unsafe {
+            sys::BwString_len(self.0)
+        }
+    }
+
+    pub fn data(&self) -> &CStr {
+        unsafe {
+            let data = sys::BwString_data(self.0);
+
+            // TODO from_bytes_with_nul_unchecked()
```

Comment:
Is this still actual? Maybe it wasn't stabilized that time.

Reply:
Done